Updates sssd to support a variety of conf parameters (#184)
plus3it/join-domain-formula: README.md

- *`install_db_dir`*: Primary directory hosting connector-software's
cache-databases (e.g., `/var/lib/pbis/db`)

### SSSD-specific pillar settings

SSSD configuration parameters for domain-specific `.conf` files under the
`/etc/sssd/conf.d/` directory can be provided using the following pillar dictionary
parameter:

- *`sssd_conf_parameters`*:
```yaml
sssd_conf_parameters   
  <key1>: '<value1>'
  ...
  <keyN>: '<valueN>'
```

> A variety of supported parameters can used to customize the operation of `sssd`.

Reviewer: ```suggestion
Any parameter supported by `sssd` may be used to customize the operation of `sssd`.
```

Author: I used your suggesting with a slight change.
